core/templates/pages/story-editor-page/editor-tab/story-editor.component.ts

```diff
@@ -366,6 +412,7 @@ export class StoryEditorComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.storyPreviewCardIsShown = false;
     this.mainStoryCardIsShown = true;
+    this.selectedChapterIndexToPublishUpToInDropdown = 0;
```

Comment:
I am not able to understand what this variable is for, maybe changing its name a bit will help

Reply:
This is the value that is selected in the publish upto chapter dropdown. Its further use will be made in the upcoming PRs.

Reply:
Then should it be `selectedChapterIndexInPublishUptoDropdown`

Reply:
Done